js/bootstrap-datepicker.js

```diff
@@ -1543,6 +1543,10 @@
 				p.setRange(range);
 			});
 		},
+		clearDates: function(){
+				this.pickers[0].clearDates();
+				this.pickers[1].clearDates();
```

Comment:
dateRange *always* has two pickers, so the assumption of 0 and 1 are safe.
If it would be more, it wouldn't be a daterange.